tools/small_rna_map/small_rna_map.py

```diff
@@ -35,28 +35,28 @@ def Ref_coord_pol(bamfile):
                         read_rname = read.reference_name
                         read_position = read.pos + 1
                         the_key = (read_rname, read_position,polarity)
+                        cigar = read.cigar[0][1]
```

Comment:
On another note: be simple, if cigar is read length call it read_length, not cigar.